challenges/functions.js

```diff
@@ -13,12 +16,30 @@
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+function consume(a,b, add) {
+  const sum = (x,y) => {
```

Comment:
I think you may have misunderstood these directions. You were supposed to create `consume()` and then create the remaining functions separately.